07-Joshua-Luo/wk01/2-tue/js/exercises.js

```diff
@@ -0,0 +1,211 @@
+// the amount is 1 million. For example:
+
+//   DrEvil(10): 10 dollars
+//   DrEvil(1000000): 1000000 dollars (pinky)
+
+function DrEvil(amount){
+    let end;
+    if (amount === 1000000){
+        end = ' (pinky)';
+    }
+    else{
+        end = '';
+    }
+    // console.log(`DrEvil(${amount}): ${amount} dollars ${end}`);
+    return (`${amount} dollars ${end}`);
+}
+
+console.log(DrEvil(1));
+console.log(DrEvil(1000000));
+// MixUp
+
+// Create a function called mixUp. It should take in two strings, and return the concatenation of the two strings (separated by a space) slicing 
+// out and swapping the first 2 characters of each. You can assume that the strings are at least 2 characters long. For example:
+
+//   mixUp('mix', 'pod'): 'pox mid'
+//   mixUp('dog', 'dinner'): 'dig donner'
+// Look up the JavaScript string reference to find methods which may be useful!
+
+function mixUp(string1,string2){
+    let len1=String(string1).length;
+    let len2=String(string2).length;
+    let newString1;
+    let newString2;
+    if(len1>=2 & len2>=2){
+        newString1=String(string2).substring(0,2)+String(string1).substring(2);
+        newString2=String(string1).substring(0,2)+String(string2).substring(2);
+    }
+    // console.log(`mixUp('${string1}', '${string2}'): '${newString1} ${newString2}'`);
+    return `${newString1} ${newString2}`;
+}
+
+console.log(mixUp(123456,654321));
+
+
+// FixStart
+
+// Create a function called fixStart. It should take a single argument, a string, and return a version where all occurences of its first character 
+// have been replaced with '*', except for the first character itself. You can assume that the string is at least one character long. For example:
+
+// fixStart('babble'): 'ba**le'
+
+function fixStart(string3){
+    let str3=String(string3);
+    let len3=str3.length;
+    let arr = "";
+    for(let i =0; i<len3; i++){
+        // let countL= arr.split(str3[i])
+        // console.log(countL);
+        if (arr.indexOf(str3[i]) === -1){
+            
+            arr=arr+str3[i];
+            // console.log(arr);
+        }
+        else{
+            
+            arr=arr+"*";
+            // console.log(arr);
+        }
+    }
+    // console.log(`fixStart('${string3}'): '${arr}'`)
+    return  arr;
+}
+console.log(fixStart('aaaabbbb'));
+
+// Verbing
+
+// Create a function called verbing. It should take a single argument, a string. If its length is at least 3, it should add 'ing' to its end, unless 
+// it already ends in 'ing', in which case it should add 'ly' instead. If the string length is less than 3, it should leave it unchanged. For example:
+
+//   verbing('swim'): 'swimming'
+//   verbing('swimming'): 'swimmingly'
+//   verbing('go'): 'go'
+
+function verbing(string4){
+    len4 = string4.length;
+    let str4;
+    if (len4<3){
+        str4=string4;
+    }
+    else{
+        if(string4.indexOf("ing") != -1){
```

Comment:
```suggestion
        if(string4.indexOf("ing") !== -1){
```
Just like the equality operatory, there is a loose and strict version of the inequality operator. We recommend using the strict version.